ui/apps/platform/src/Containers/Clusters/InitBundles/SecureClusterUsingHelmChart.tsx

```diff
@@ -8,15 +30,95 @@ export type SecureClusterUsingHelmChartProps = {
 function SecureClusterUsingHelmChart({
     headingLevel,
 }: SecureClusterUsingHelmChartProps): ReactElement {
+    const { version } = useMetadata();
+    const subHeadingLevel = headingLevel === 'h2' ? 'h3' : 'h4';
+    const [wasCopied, setWasCopied] = useState(false);
+
+    function onClickCopy() {
+        // https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText#browser_compatibility
+        // Chrome 66 Edge 79 Firefox 63 Safari 13.1
+        navigator?.clipboard
+            ?.writeText(codeBlock)
+            .then(() => {
+                setWasCopied(true);
+            })
+            .catch(() => {
+                // TODO addToast(title, message)
+            });
+    }
+
+    const actions = (
+        <CodeBlockAction>
+            <ClipboardCopyButton
+                aria-label="Copy to clipboard"
+                id="ClipboardCopyButton"
+                onClick={onClickCopy}
+                textId="CodeBlockCode"
+                variant="plain"
+            >
+                {wasCopied ? 'Copied to clipboard' : 'Copy to clipboard'}
+            </ClipboardCopyButton>
+        </CodeBlockAction>
+    );
+
     return (
         <Flex direction={{ default: 'column' }}>
             <Title headingLevel={headingLevel}>
                 Secure a cluster using Helm chart installation method
             </Title>
+            {version && (
+                <ExternalLink>
+                    <a
+                        href={getVersionedDocs(
+                            version,
+                            '/installing/installing_other/install-secured-cluster-other.html'
+                        )}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Installing secured cluster services on other platforms
+                    </a>
+                </ExternalLink>
+            )}
+            <Title headingLevel={subHeadingLevel}>Prerequisites</Title>
+            <List component="ol">
+                <ListItem>
+                    <p>You must add the Helm chart repository. TODO is this step 2?</p>
```

Comment:
This step only needs to be done once (ever, I believe) on each new system. It is done with the `helm repo add rhacs https://mirror.openshift.com/pub/rhacs/charts/` command. So if this is a prerequisite, it could be something like:

You must have previously added the Helm chart repository, or add it using the `helm repo add rhacs https://mirror.openshift.com/pub/rhacs/charts/` command.

Please verify but this is my understanding of the situation.

Reply:
Simon, can you verify?